src/cmd-buildextend-live

```diff
@@ -524,6 +524,15 @@ def generate_iso():
             if len(vendor_ids) != 1:
                 raise Exception(f"did not find exactly one EFI vendor ID: {vendor_ids}")
 
+            # scos: rename EFI/{distro} to EFI/centos
+            # https://github.com/openshift/os/issues/954
+            if base_name == "scos":
```

Comment:
Since we discovered the correct vendor id here (`vendor_id[0]`), we could also just do this by default across the board I think.

If we did that, then in our config git repos, we could have it be `live/EFI/vendor/grub.cfg` - i.e. it *always* gets renamed.

A benefit of this is that we have one less distribution-specific tweak in coreos-assembler.  These often tend to get stale.

Here's one other example in this file:

```
if base_name == "rhcos" and args.fast:
    print("'--fast' requires LZ4 compressed squashfs support in the kernel. This is \
currently only available for FCOS")
    sys.exit(1)
```

This code predates RHEL9 - and is now incorrect there.  What we should have probably been doing is checking the kernel config file in `/usr/lib/modules/$kver/config`.

Reply:
That makes sense. Does this mean I should rename current `live/EFI/redhat/grub.cfg` to `live/EFI/vendor/grub.cfg` in [os repo](https://github.com/openshift/os/tree/master/live/EFI/redhat) with this PR?

>This code predates RHEL9 - and is now incorrect there. What we should have probably been doing is checking the kernel config file in /usr/lib/modules/$kver/config

Checked in https://gitlab.com/redhat/centos-stream/rpms/kernel/-/blob/c9s/kernel-x86_64-rhel.config#L5382 (# CONFIG_SQUASHFS_LZ4 is not set), this is not enabled in C9S yet, should we keep as it is?

Reply:
> That makes sense. Does this mean I should rename current live/EFI/redhat/grub.cfg to live/EFI/vendor/grub.cfg in [os repo](https://github.com/openshift/os/tree/master/live/EFI/redhat)? (And need to do change to this PR)

In my proposal the name of the directory in the config git repository wouldn't matter; we'd automatically rename it to whatever the target system's EFI vendor directory is.  It would be cleaner to rename it to `vendor`, but not required.

(Just to be 100% clear I am good to merge this PR as is, if you agree please go ahead and merge - this adjustment can happen as a followup)